### PR TITLE
Created test file 'test.md', page name is Test

### PR DESCRIPTION
**Pull request by xiaoeva**
I created a test file called test.md

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+--
+layout: page
+title: Test
+permalink: ?/test/
+--
+
+## Testing
+
+Content content
```
Comment (emj365) [inline test.md]: Wrong

Comment (xiaoeva) [inline test.md]: YOU'RE wrong!!!

Comment (lpgn) [inline test.md]: is this a competition? ;-)

Comment (emj365) [inline test.md]: You're right :smiley:

**Pull request closed without merging**